recipes/teemo/all/conandata.yml

```diff
@@ -1,7 +1,7 @@
 sources:
   "2.7":
     url: "https://github.com/winsoft666/teemo/archive/refs/tags/v2.7.tar.gz"
-    sha256: "97be8ae62cd3e2e92ed0ff405dfaf5e6cb6e323b21c53e798adfa5872a2f7084"
```

Comment:
Why has the checksum changed?

Reply:
Sorry, I don't know why.

Reply:
Just found out they are redirecting the files to the new zoe location, and the name changed. Otherwise seems identical, so approving now, thanks a lot for your patience on this one :)